doc/lua_api.txt

```diff
@@ -3554,14 +3554,12 @@ Utilities
           avg_jitter = 0.03,         -- average packet time jitter
           connection_uptime = 200,   -- seconds since client connected
           protocol_version = 32,     -- protocol version used by client
-          -- following information is available on debug build only!!!
-          -- DO NOT USE IN MODS
-          --ser_vers = 26,             -- serialization version used by client
-          --major = 0,                 -- major version number
-          --minor = 4,                 -- minor version number
-          --patch = 10,                -- patch version number
-          --vers_string = "0.4.9-git", -- full version string
-          --state = "Active"           -- current client state
+          serialization_version = 26,             -- serialization version used by client
+          major = 0,                 -- major version number
+          minor = 4,                 -- minor version number
+          patch = 10,                -- patch version number
+          version_string = "0.4.9-git", -- full version string
```

Comment:
This needs updating - most of these fields are now unavailable in Release builds again.